docs/v3-ea-components/transports.md

```diff
@@ -17,21 +17,8 @@ If you find that the built-in features of a transport do not meet your endpoint'
 - [SSE Transport](./transport-types/sse-transport.md)
 - [Custom Transport](./transport-types/custom-transport.md)
 
-## Routing Transport
+### Meta Transports
 
-As mentioned above, each endpoint only accepts a single transport. However, there are scenarios where a data provider will have multiple ways to access the same data. For example, a data provider could offer crypto data through REST API and Websockets which would require two different transports. The `RoutingTransport` allows an endpoint to have access to both.
+In some cases, a combination of Transports is necessary. For example, an EA endpoint might be reliant on a combination of transports.
```

Comment:
This is me just nitpicking but this might sound a little redundant. Could we combine these sentences? Or rephrase one of them?
```suggestion
In some cases, an EA endpoint might be reliant on a combination of transports.
```